_posts/2021-02-15-fix-intermittently-failing-ci-builds-flaky-tests-rspec.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Fix intermittently failing CI builds due to flaky tests in RSpec"
+date:   2021-02-15 12:00:00 +0100
+author: "Shadi Rezek"
+categories: continuous_integration techtips rspec
+og_image: "/images/blog/posts/fix-intermittently-failing-ci-builds-flaky-tests-rspec/flaky_tests.jpeg"
+---
+
+I hate randomly failing CI builds. One run is green, another is red (with no changes in the code!). So frustrating. Let's see what can be done with this.
+
+<img src="/images/blog/posts/fix-intermittently-failing-ci-builds-flaky-tests-rspec/flaky_tests.jpeg" style="width:150px;margin-right: 15px;float:left;" alt="CI Builds, Flaky Tests, Knapsack Pro" />
```

Comment:
I guess you added an image on the left side to read it along with the header as `Flaky tests Random failures` 

Maybe a bit bigger size of the image would be more eye catching.

Reply:
Sorry, I don't understand what you mean with the header. Can you elaborate?

I based the whole `<img>` tag on another, already published blog post. I thought it's like that so that the logo is on the left in the index view.

I realized I forgot to check how it looks in development. I'll do it on Monday

Reply:
`Flaky tests Random failures` - this is me overthinking. Sorry. When you open a blog post the image is on the left side next to the header `Random failures` so that's why I read it like one sentence. 

In most cases, the image is on the right side  in the blog content so it won't affect the alignment of headers.
For instance here https://docs.knapsackpro.com/2019/how-to-use-codefresh-ci-parallel-steps-to-run-rspec-a-few-times-faster-for-rails-project the image is on the right side so the header `Configure Rails project on Codefresh.io` is aligned with the text.

When you go to list of all articles then the image is on the left side but it's like that because it's read from `og_image` tag. Not from `<img>` tag from the blog content.

Reply:
I see, thanks for the suggestions. It looks better on the right.